.storybook/manager-head.html

```diff
@@ -0,0 +1,103 @@
+<style type="text/css">
+    /* styles for the storybook sidebar */
+    .search-result-item {
+        border-bottom: 1px solid rgba(0, 0, 0, 0.1) !important;
```

Comment:
are all those `!important` needed? There isn't any official way of customizing these storybook styles?

I'm afraid a future update of storybook could break this

Reply:
I followed the example from NASA storybook
https://github.com/nasa-jpl/explorer-1/blob/main/storybook/manager-head.html

https://storybook.js.org/docs/react/configure/theming#css-escape-hatches

Reply:
I removed `!important` tag from the most of cases. The cases with `!important` needs it to force override the styles :(